use-cases/virgo/config.yaml

```diff
@@ -1,7 +1,8 @@
-root_folder: /p/scratch/intertwin/datasets/virgo
+hdf5_file_location: /p/scratch/intertwin/datasets/virgo_hdf5/virgo_data.hdf5
+hdf5_dataset_name: virgo_dataset
 data_root: ./data
-epochs: 2
-batch_size: 3
+epochs: 5
+batch_size: 2000 # equivalent to chunk size 
```

Comment:
are you sure that you can comment inline?

Reply:
As in whether it crashes or not or as in whether it is best practice?

Reply:
whether it crashes. That's sth you cannot do with a workflow files for GH actions, as it will consider the whole line as a string, inclluding the comment (as far as I remember). In this case the YAML parser may be smarter though, as it expects an int.

In general it does not look like a good practice, although I may also have been doing it in these files.